egs/apiai_decode/s5/local/score.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+# Copyright 2012-2014  Johns Hopkins University (Author: Daniel Povey, Yenda Trmal)
+# Apache 2.0
```

Comment:
It would be more straightforward, I think, to make local/score.sh a soft link to (IIRC) ../steps/score_kaldi.sh.  See if that works for you.